lib/csv.rb

```diff
@@ -1464,6 +1464,46 @@ def generate_line(row, **options)
       (new(str, **options) << row).string
     end
 
+    # :call-seq:
+    #   CSV.generate_lines(rows)
+    #   CSV.generate_lines(rows, **options)
+    #
+    # Returns the \String created by generating \CSV from
+    # using the specified +options+.
+    #
+    # Argument +rows+ must be an \Array of row. Row is \Array of \String or \CSV::Row.
+    #
+    # Special options:
+    # * Option <tt>:row_sep</tt> defaults to <tt>"\n"> on Ruby 3.0 or later
+    #   and <tt>$INPUT_RECORD_SEPARATOR</tt> (<tt>$/</tt>) otherwise.:
+    #     $INPUT_RECORD_SEPARATOR # => "\n"
+    # * This method accepts an additional option, <tt>:encoding</tt>, which sets the base
+    #   Encoding for the output. This method will try to guess your Encoding from
+    #   the first non-+nil+ field in +row+, if possible, but you may need to use
+    #   this parameter as a backup plan.
+    #
+    # For other +options+,
+    # see {Options for Generating}[#class-CSV-label-Options+for+Generating].
+    #
+    # ---
+    #
+    # Returns the \String generated from an
+    #   CSV.generate_lines(['foo', '0'],['bar', '1'],['baz', '2']) # => "foo,0\r\nbar,1\r|nbaz.2"
```

Comment:
```suggestion
    #   CSV.generate_lines(['foo', '0'], ['bar', '1'], ['baz', '2']) # => "foo,0\r\nbar,1\r\nnbaz,2\r\n"
```

BTW, is `\r\n` really used for row separator? The above document says "Option `:row_sep` defaults to `"\n"`.

Reply:
In the `general_line` method, it is written like this `CSV.generate_line(['foo', '0']) # => "foo,0\n"` . So I think that I should write the same as this.